java/server/src/org/openqa/selenium/remote/server/DriverServlet.java

```diff
@@ -98,7 +98,7 @@ protected void createSessionCleaner(Logger logger, DriverSessions driverSessions
 
   private long getValueToUseInMs(String propertyName, long defaultValue) {
     long time = defaultValue;
-    final String property = getServletContext().getInitParameter(propertyName);
+    final String property = System.getProperty(propertyName);
```

Comment:
This field isn't always specified on the command line, this will break the current server:

https://github.com/SeleniumHQ/selenium/blob/master/java/server/src/org/openqa/selenium/server/SeleniumServer.java#L414-L416